docs/documentation/configuration/index.md

```diff
@@ -227,3 +227,35 @@ unnecessary in a testing environment.
 
 The search path for global modules. Almost always unnecessary in a testing
 environment.
+
+### disable_module_hiera
+**Type:** Boolean<br /> 
+**Default:** `false`<br />
+**Puppet Version(s):** 6.x
```

Comment:
I believe these features should work with Puppet version >= 4.9.0, not just 6.x

Reply:
adjusted for all new config entries